### PR TITLE
[DBD-91] Update Commit Messages Post

### PR DESCRIPTION
**Pull request by mzaeemz**

### CODE REVIEW

File: _posts/2020-03-26-commit-messages.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ There is a task ID assigned to each task attempted. We do it normally through Tr
 
 Assuming we have a task in a project named "My Good Project", it will be assigned a three-letter code (by convention) followed by a hyphen and a task number. Such as, `MGP-1`.
 
-`MGP-1` will be prepended with commit message wrapped in square brackets as
+`MGP-1` will be prepended with pull request wrapped in square brackets as
```
Comment (fahadsiddiqui) [inline _posts/2020-03-26-commit-messages.md:23]:
```suggestion
`MGP-1` will be prepended with pull request title, wrapped in square brackets as
```

File: _posts/2020-03-26-commit-messages.md [hunk 2]
```diff
@@ -57,34 +58,33 @@ smthing
...
 ```
 
 > "Playing my part."
 
 ```
-[MGP-1] .DS_Store Added in .gitignore
+.DS_Store Added in .gitignore
 ```
 
 ## What Makes Commit Title, Perfect?
```
Comment (fahadsiddiqui) [inline _posts/2020-03-26-commit-messages.md:84]:
```suggestion
## Summary
```

File: _posts/2020-03-26-commit-messages.md (hunk 1 above)
Comment (fahadsiddiqui) [inline _posts/2020-03-26-commit-messages.md:23]:
```suggestion
`MGP-1` will be prepended with pull request title, wrapped in square brackets as
```

File: _posts/2020-03-26-commit-messages.md (hunk 2 above)
Comment (fahadsiddiqui) [reply _posts/2020-03-26-commit-messages.md:84]: Make this change in whole post.

**Review by fahadsiddiqui — CHANGES_REQUESTED** [review]

File: _posts/2020-03-26-commit-messages.md (hunk 2 above)
Comment (mzaeemz) [reply _posts/2020-03-26-commit-messages.md:84]: Should I add "Commit title should have: " below Summary heading above bulleted list?

**Review by mzaeemz — COMMENTED** [review]
Changes made, kindly review.

**Review by fahadsiddiqui — APPROVED** [review]

**Pull request merged**